docs_src/design/adr/devops/0007-Release-Automation.md

```diff
@@ -0,0 +1,87 @@
+# Release Automation
+
+## Status
+
+In Review for Geneva
+
+## Context
+
+EdgeX Foundry is a framework composed of microservices to ease development of IoT/Edge solutions. With the framework getting richer, project growth, the number of artefacts to be released has increased. This proposal outlines the automation of the release process for the base artifacts.
```

Comment:
number of **artefacts** --> artifacts

Reply:
suggest: _This proposal outlines a method for automating the release process for the base artifacts._